frontend/src/bundles/recently-viewed/recently-viewed-api.ts

```diff
@@ -0,0 +1,40 @@
+import { ApiPath, ContentType } from '~/bundles/common/enums/enums.js';
+import { HttpApi } from '~/framework/api/api.js';
+import { type IHttp } from '~/framework/http/http.js';
+import { type IStorage } from '~/framework/storage/storage.js';
+
+import { RecentlyViewedApiPath } from './enums/enums.js';
+import { type RecentlyViewedTemplatesResponseDto } from './types/types.js';
+
+type Constructor = {
+    baseUrl: string;
+    http: IHttp;
+    storage: IStorage;
+};
+
+class RecentlyViewedApi extends HttpApi {
+    public constructor({ baseUrl, http, storage }: Constructor) {
+        super({ path: ApiPath.RECENTLY_VIEWED, baseUrl, http, storage });
+    }
+
+    public async getRecentlyViewedTemplates(
+        limit = '10',
```

Comment:
Consider replacing with a constant

Reply:
I'm not sure it's good idea. I use it as default params

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-resumegemm/assets/92567020/334e6377-d038-45f8-9f67-56931684de5c)

https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md

Reply:
I know what you mean. but it's param not variable. I'll ask about it))